cpp/cmake/Modules/JitifyPreprocessKernels.cmake

```diff
@@ -16,7 +16,7 @@
 add_executable(jitify_preprocess "${JITIFY_INCLUDE_DIR}/jitify2_preprocess.cpp")
 
 target_compile_definitions(jitify_preprocess PRIVATE "_FILE_OFFSET_BITS=64")
-target_link_libraries(jitify_preprocess CUDA::cudart ${CMAKE_DL_LIBS})
+target_link_libraries(jitify_preprocess CUDA::cudart${_ctk_static_suffix} ${CMAKE_DL_LIBS})
```

Comment:
Why does this need to remain (why isn't `rapids_cuda_init_runtime` setting `CMAKE_CUDA_RUNTIME_LIBRARY` not enough) when you are able to remove the explicit `target_link_libraries` for cudf above?